README.md

```diff
@@ -1,4 +1 @@
-Roses are red, 
-violets are blue.
-Sugar is sweet, 
-and so are you :)
+Pretend there's another poem here...
```

Comment:
This is not very helpful but I'll give you a pass...for...now.

Reply:
mean :(